internal/prototest/examplepb/example.proto

```diff
@@ -45,6 +45,14 @@ message EchoBothResponse {
   string message = 1;
 }
 
+message Get2ndMessageRequest {
+   string message = 1;
+}
+
+message Get2ndMessageResponse {
+  string message = 1;
+}
+
```

Comment:
Please add a comment on the message on why we decided to add this type of message name.